frontend/src/pages/Sessions/EmptySessionsSearchParams.tsx

```diff
@@ -19,5 +19,5 @@ export const EmptySessionsSearchParams: Complete<SearchParamsInput> = {
 	environments: [],
 	app_versions: [],
 	show_live_sessions: false,
-	query: `{\"isAnd\":true,\"rules\":[[\"custom_processed\",\"is\",\"true\"]]}`,
+	query: `{\"isAnd\":true,\"rules\":[]}`,
```

Comment:
Can we make it an empty string?
![Screenshot 2023-03-08 at 12 50 47 PM](https://user-images.githubusercontent.com/17913919/223790937-65071fd9-7e46-485f-9528-40bb68e7983b.jpg)

Reply:
updated